frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template.tsx

```diff
@@ -42,6 +42,7 @@ import { menuActions } from './menu-actions';
 import { TemplateSource } from './vm-template-source';
 
 import './vm-template.scss';
+import { withStartGuide } from '@console/internal/components/start-guide';
```

Comment:
move this import above `scss` and local imports please (line 31 and above)

Reply:
Done!